### DISCUSSION TITLE
Add `max-h-none` utility

### DISCUSSION THREAD

**Discussion by MichaelAllenWarner** [tailwindlabs/tailwindcss#6030] [Ideas]
Although `none` is already the initial value for the `max-height` property, having a `max-h-none` utility available would be useful so that another `max-height` value can be _undone_ at some breakpoint, like `max-h-40 lg:max-h-none`.

**Comment by mamunonweb** [comment]
The current solution is `max-h-[none]`. But yes, I think we should have a `max-h-none`.
